<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewJob</class>
 <widget class="QMainWindow" name="NewJob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Legg til en ny jobb</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditTittel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Tittelen til den nye jobben</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBedrift">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Bedrift</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStatusID">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSted">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSoknadsfrist">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Dato, kan være i hvilket format som helst.</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Her kan du legge inn en ny søknad i databasen. Alle felt må fylles ut.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelTittel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Tittel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStedID">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>166</width>
      <height>22</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatusID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>StatusID</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStedid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Stedid</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSoknadsfrist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Søknadsfrist</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBedrift">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Hva heter bedriften?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditTittel</tabstop>
  <tabstop>lineEditBedrift</tabstop>
  <tabstop>comboBoxStedID</tabstop>
  <tabstop>comboBoxStatusID</tabstop>
  <tabstop>lineEditSoknadsfrist</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
